todo_app/templates/index.html

```diff
@@ -11,8 +11,21 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        <!-- List the todo items here --> 
+
+          {% for item in get_items() %}
+          
+            <li>{{ item.title }}</li>
+        
+          {% endfor %}
+        
       </ul>
+      <!-- setting the URL path for where the form is to be 'posted' to -->
+      <form action="/additem" method="post">
+        <input type="text" name="title" class="form_input">
```

Comment:
Minor, but it might be nice to have a `required` attribute on the input to prevent us adding empty tasks